woocommerce/woocommerce-admin
# Fix some data not to being imported when 'skip_existing' option is enabled

Fixes #2343.

When `skip_existing` was set to true, in every batch the table of customers/orders to import was getting smaller (because recently imported customers/orders were not included) but we were still increasing the `offset` in each query. That made last batches to import nothing.

Making sure `offset` is always `0` when `skip_existing` is true, fixes the issue.

### Detailed test instructions:

I found it easier to work on this merging the branch from #2312. That way, you get progress bars in the import tool and you can see how many customers/orders are imported every time.

- Go to _Analytics_ > _Settings_.
- Click on _Delete Previously Imported Data_ and make sure all tasks run.
- Then, start an import making sure you have _Skip previously imported customers and orders_ selected.
- Verify it imports all customers and all orders.
